opencti-platform/opencti-front/src/private/components/analyses/reports/ReportEditionOverview.jsx

```diff
@@ -329,6 +329,7 @@ const ReportEditionOverviewComponent = (props) => {
             helpertext={
               <SubscriptionFocus context={context} fieldname="objectMarking" />
             }
+            setFieldValue={setFieldValue}
```

Comment:
Need to be add in all editions form

Reply:
It will be added later :)
For the PR, the changes is made only for Report at first for testing.